src/music.h

```diff
@@ -23,6 +23,9 @@
 #ifndef MUSIC_H_
 #define MUSIC_H_
 
+#define LOOP_DEBUG_PRINTOUT 1
+/*#define LOOP_DEBUG_PRINTOUT 0*/
```

Comment:
You don't need to keep this here. Please make the CMake side option to turn this on/off :eyes:

Reply:
👍